docs/deploying-to-azure.md

```diff
@@ -205,7 +207,334 @@ az postgres flexible-server show-connection-string \
   --query connectionStrings.jdbc
 ```
 
+### Create the managed MongoDB Database
+
+We need to create a MongoDB so the Fight microservice can store data.
+Create a database in the region where it's available:
+
+```shell
+az cosmosdb create \
+  --resource-group $RESOURCE_GROUP \
+  --locations regionName="$LOCATION" failoverPriority=0 \
+  --name $MONGO_DB \
+  --kind MongoDB \
+  --server-version 4.0
+```
+
+Create the Fight collection:
+
+````shell
+az cosmosdb mongodb database create \
+  --resource-group $RESOURCE_GROUP \
+  --account-name $MONGO_DB \
+  --name $FIGHTS_DB_SCHEMA
+````
+
+The Book microservice communicates with the Book Fail microservice through Kafka.
+We need to create an Azure event hub for that.
+
+```shell
+az eventhubs namespace create \
+  --resource-group $RESOURCE_GROUP \
+  --location $LOCATION \
+  --name $EVENTHUB_NAMESPACE
+```
+
+```shell
+az eventhubs eventhub create \
+  --resource-group $RESOURCE_GROUP \
+  --name $EVENTHUB_TOPIC \
+  --namespace-name $EVENTHUB_NAMESPACE
+```
+
+### Create the Managed Kafka
+
+The Fight microservice communicates with the Statistics microservice through Kafka.
+We need to create an Azure event hub for that.
+
+```shell
+az eventhubs namespace create \
+  --resource-group $RESOURCE_GROUP \
+  --location $LOCATION \
+  --name $KAFKA_NAMESPACE
+```
+
+```shell
+az eventhubs eventhub create \
+  --resource-group $RESOURCE_GROUP \
+  --name $KAFKA_TOPIC \
+  --namespace-name $KAFKA_NAMESPACE
+```
+
 ## Deploying the Application
 
-### The Heroes Microservice
+Now that the Azure Container Apps environment is all set, we need to configure our microservices, build them as Docker images, push them to Docker Hub, and deploy these images to Azure Container Apps.
+So let's configure, build, push and deploy each Microservice.
+
+### Heroes Microservice
+
+The Heroes microservice needs to access the managed Postgres database.
+Therefore, we need to set the right properties in the file `application-azure.yml`.
+We know the values for the Postgres database, admin and password (our `POSTGRES_DB`, `POSTGRES_DB_ADMIN` and `POSTGRES_DB_PWD` variables), so the configuration looks like this:
+
+```shell
+quarkus:
+  datasource:
+    username: superheroesadmin
+    password: super-heroes-p#ssw0rd-12046
+    reactive:
+      url: postgresql://super-heroes-db.postgres.database.azure.com:5432/heroes_database?ssl=true&sslmode=require
+```
+
+We build the Heroes microservice with the following command:
```

Comment:
Why do we need to build the image? The image is already there and produced by the super heroes CI.